packages/suite/src/support/messages.ts

```diff
@@ -4202,14 +4202,6 @@ export default defineMessages({
         id: 'TR_EXCHANGE',
         defaultMessage: 'Exchange',
     },
-    TR_PASSWORDS: {
-        id: 'TR_PASSWORDS',
-        defaultMessage: 'Passwords',
-    },
-    TR_PORTFOLIO: {
-        id: 'TR_PORTFOLIO',
-        defaultMessage: 'Portfolio',
-    },
```

Comment:
We probably should remove them also from the files for other languages like `/packages/suite-data/files/translations/cs.json' and rest of languages.

Reply:
thats gonna get auto-regenerated imho during next sync

Reply:
Ahh ok, thanks for the clarification.